### PR TITLE
Updated to add proxy switches and reed switches

### PR DESCRIPTION
**Pull request by Beckermi**
First time using Github.. hoping I can access / create the pages for these two switch types in order to write about them and upload pics..

### CODE REVIEW

File: mechs/switches/index.rst [hunk 1]
```diff
@@ -16,11 +16,17 @@ MPF's *switch* device represents a switch in a pinball machine. This device is
...
 
-There are two general switch types in pinball machines (read those for details):
+There are two switch types most commonly seen in pinball machines (read those for details):
 
 * :doc:`Mechanical switches <mechanical_switches>`
 * :doc:`Optical switches <optos>`
 
+And an additional two types used in a handful of machines (read those for details):
+
+* :doc: 'Proximity switches <Proxy_switches>'
```
Comment (jabdoa2) [inline mechs/switches/index.rst]: Almost. The space after `:doc: ` needs to go. I'm not sure if you can create new pages in the web gui though. I can do that for you if you want.

Comment (Beckermi) [reply mechs/switches/index.rst]: ok, I think it's fixed.  If you can create those blank pages I can add pictures and some info on them, that would be great while I get up to speed on GitHub

Comment (jabdoa2) [reply mechs/switches/index.rst]: Just checked. You can click "create new file" or "upload file" here: https://github.com/Beckermi/mpf-docs/tree/patch-1/mechs/switches

**Comment by jabdoa2** [comment]
I pushed some small fixes. Let me know when you are ready and we will merge your changes.

**Comment by Beckermi** [comment]
That looks good, once the page is created then I should be able to go back
and add the pictures. Sorry  about my slow learning on GitHub.. still over
my head but I'll get there...

**Comment by jabdoa2** [comment]
Merged this manually

**Pull request closed without merging**